vividus-extension-selenium/src/main/java/org/vividus/bdd/steps/ui/GenericElementSteps.java

```diff
@@ -77,4 +90,45 @@ public void assertElementsNumberInState(State state, Locator locator, Comparison
         assertElementsNumber(locator, comparisonRule, quantity)
                 .forEach(e -> baseValidations.assertElementState(description, state, e));
     }
+
+    /**
+     * Verifies if the elements located by the <b>locator</b> are sorted in <b>sortingOrder</b> order by their text
+     * @param locator locator to locate elements
+     * @param sortingOrder sorting order, can be either <b>ASCENDING</b> or <b>DESCENDING</b>
+     */
+    @Then("elements located `$locator` are sorted by text in $sortingOrder order")
+    public void areElementSorted(Locator locator, StringSortingOrder sortingOrder)
+    {
+        ComparisonRule rule = ComparisonRule.GREATER_THAN;
+        int requiredNumber = 1;
+
+        List<WebElement> elements = baseValidations.assertNumberOfElementsFound("The elements to check the sorting",
+                locator, requiredNumber, rule);
+
+        Matcher<Integer> requiredSizeMatcher = rule.getComparisonRule(requiredNumber);
+        if (requiredSizeMatcher.matches(elements.size()))
```

Comment:
just size() > 1

Reply:
Why should we consider a single element as a fail?

Reply:
from end users side it makes sense to sort more than one element, on the other hand, if we sort even one element, empty collections should be also allowed because this is not prohibited from programatic side